Week-2/Daily-Assignments/Assignment2/tempCodeRunnerFile.ts

```diff
@@ -0,0 +1 @@
+ JSON.parse(response)
```

Comment:
where is this file used?

Reply:
it was accidentally created by code runner extension. ill remove it